composer.json

```diff
@@ -15,7 +15,7 @@
 	],
 	"minimum-stability": "dev",
 	"require": {
-		"php": ">=5.4"
+		"php": ">=7.4"
```

Comment:
I wonder is 7.4 a real requirement within the codebase of this plugin? 7.0-7.3 still take more than 20% of setups according to [WordPress Stats](https://wordpress.org/about/stats/)

Reply:
@cadic I imagine 7.4 was taken from the linked issue #93 which is what we want to bump our plugin minimums to at this point

Reply:
@cadic that sound ok to you or should we stick with 7.0?